.github/workflows/documentation.yml

```
@@ -22,7 +22,9 @@ jobs:
         uses: actions/cache@v4
         with:
           path: ./.venv
-          key: venv-${{ hashFiles('poetry.lock') }}
+          key: venv-${{ runner.os }}-${{ hashFiles('**/poetry.lock') }}
+          restore-keys: |
+            venv-${{ runner.os }}-
```

Let's change this back 

```suggestion
          key: venv-${{ hashFiles('poetry.lock') }}
```